extensions/adobe/experience/adcloud/addeliverydetails.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/addeliverydetails",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Advertising Cloud Search Ad Delivery Details",
+  "type": "object",
+  "description": "Adobe Advertising Cloud Search Ad Delivery Details.",
+  "definitions": {
+    "adDeliveryDetails": {
+      "properties": {
+        "xdm:adPosition": {
```

Comment:
There is another ad cloud PR (#648) that ads a slot and position to the "search" element in XDM core.  How would these two relate?  Can they be one field, or at least use the same name in order to show they mean the same thing semantically?

Reply:
CC @sathyz who filed #648

Reply:
Kevin, yes, we will remove adposition from this extension and will use the changes in #648